core/jazz_es-kinesis-log-streamer/config/global_config.json

```diff
@@ -12,7 +12,7 @@
     "lambda_request_id": "^END RequestId: (.+\n)$",
     "lambda_environment": "^/aws/lambda/(.+)-(.+)$",
     "lambda_environment_dev": "^/aws/lambda/(.+)-(.*\\b(-dev|-stg|-prod))$",
-    "lambda_domain_service": "^/aws/lambda/(.+)-(.+)$",
+    "lambda_domain_service": "^/aws/lambda/{stack_prefix}-(.+)-(.+)$",
```

Comment:
Shouldn't we look for same patterns for `lambda_environment_dev` & `lambda_environment` above?

Reply:
`lambda_environment` is working as expected. Updated `lambda_environment_dev`